src/ResourceManagement/Logic/Microsoft.Azure.Management.Logic/project.json

```diff
@@ -1,5 +1,5 @@
 ﻿{
-  "version": "1.0.0",
+  "version": "2.0.0",
```

Comment:
@pankajsn can you do same version bump in AssemblyInfo.cs in order to be consistent
otherwise looks good.